Move 1

spookyWalrus opened the pull request:
This PR clears AB square.

spookyWalrus committed:
Clear AB

mentor-tara commented:
Looks like something is wrong :astonished:	Please see the CI checks below for more information. Click the "Details" link for more information as shown in this screenshot.

![Screenshot of CI checks](https://media.graphcms.com/qz1AmyOsQWCWNh8kcuFw)[](error)

spookyWalrus committed:
Addressed suggested edit

Changed coordinates of move to correct syntax: second character of coordinates changed to be a number.

mentor-tara reviewed (approved):
### You're good. No mines hit. Merge this PR to continue the game.

This is the current status:

|   | **A** | **B** | **C** | **D** | **E** | **F** | **G** | **H** |
|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
| **1** |   |   |   |   |   |   |   |   |
| **2** |   |   |   |   | 2 | 2 | 3 | 2 |
| **3** |   |   |   | 2 | 1 | 0 | 0 | 0 |
| **4** | 1 | 2 | 1 | 1 | 0 | 0 | 0 | 0 |
| **5** | 0 | 0 | 0 | 0 | 0 | 0 | 1 | 1 |
| **6** | 0 | 0 | 0 | 1 | 1 | 1 | 2 |   |
| **7** | 0 | 1 | 1 | 2 |   |   |   |   |
| **8** | 0 | 1 |   |   |   |   |   |   |

Mines remaining: 10
Moves made: 1[](clear-A8)
